bifunctors.cabal

```diff
@@ -54,42 +39,27 @@ flag tagged
 
 library
   hs-source-dirs: src
-  include-dirs: include
-  includes: bifunctors-common.h
   build-depends:
-    base                >= 4.3   && < 5,
+    base                >= 4.9   && < 5,
     base-orphans        >= 0.8.4 && < 1,
```

Comment:
This is used only in `Data.Bitraversable`. Would it make sense to move into `!impl(ghc > 8.2)` conditional?

Reply:
Yes, I agree. Done.